source/elements/oneMKL/source/architecture/other_architecture.inc.rst

```diff
@@ -11,10 +11,40 @@ This section covers all other features in the design of oneMKL architecture.
 
 .. _onemkl_spec_current_version:
 
-Current Version of this oneMKL Specification
+Specification Version and Compliance
 +++++++++++++++++++++++++++++++++++++++++++++
 
-This is the oneMKL specification which is part of the oneAPI specification version 1.0.0.
+Each oneMKL domain must define a preprocessor macro to represent the version of the specification that the implementation is compliant with.
+
+The macros for each domain are listed as follows:
+
+  | ONEMKL_BLAS_SPEC_VERSION
```

Comment:
If/When we switch to oneMATH name for the Specification and Interfaces project, will we also switch the macro names ?  maybe we want to delay this merge until those are made to not have to deprecate/switch ...  ?

Reply:
@spencerpatty The changes you mentioned are from https://github.com/oneapi-src/oneMKL/pull/564 which is an RFC in one of the implementations of this spec. The implementation can choose to take any direction, whether or not it is in compliance with the specification.

@Rbiessy you approved this PR earlier so there are 2 questions - 

1. Does your RFC(https://github.com/oneapi-src/oneMKL/pull/564) block this RFC?
2. Does your RFC aim to make changes to the oneAPI specification eventually? If yes, can you please create a placeholder Issue/PR in this project and mention it here? If not, please note that your oneMKL Interfaces RFC breaks compliance with the oneAPI specification as of the current status of your RFC.

Reply:
I had an offline discussion with the team and we came to an agreement that the `oneMATH` proposal has a lot of work ahead of it, and that name change will have to be made in not just oneMKL Interfaces, but also in multiple places across oneAPI spec and other documents. Since this change will have a wider impact than just this specific part of document where this PR targets, it is better to make that change at that time. It will not affect the core version schema or approach proposed here.
This PR and RFC is NOT blocked by @Rbiessy 's RFC in oneMKL Interfaces.

Reply:
It does not matter to me which RFC is implemented first. The RFC for renaming is planning to rename the occurrences of all the macros using `ONEMKL`.
The RFC describes how it impacts the specification. I read the ratification process, it does not say that an issue needs to be created. @mkrainiuk could this be clarified in the RFC process? I created https://github.com/uxlfoundation/oneAPI-spec/issues/580 but I have some concerns the discussions will be split in 2 different places now.

Reply:
Hi @Rbiessy, good point, we need to clarify it in the ratification process, and I think any changes related to API should be discussed in Spec only, since opensource is just an implementation of the Spec, I believe it should just follow what was decided for Spec. In this particular case I guess the problem is that we have opensource project migration (which is not related to the Spec) combined with renaming (which is Spec related and should be started in the Spec).

Reply:
Thank you for clarifying, @mkrainiuk . @spencerpatty @Rbiessy Let's proceed with this RFC with the `oneMKL` naming. When the `oneMATH` proposal is made to the oneAPI spec, it can address these macros and other changes across the oneAPI spec simultaneously.

Reply:
The name transition and corresponding changes from this proposal are part of https://github.com/uxlfoundation/oneAPI-spec/pull/596 now.